app/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
     implementation 'org.commcarehq.commcare:support-library:12.4'
+    implementation 'com.google.code.gson:gson:2.8.7'
```

Comment:
@zandre-eng is this library part of the fix? Also, I remember that this dependency had already been added to the project

Reply:
This is not directly related to the input param fix, however I've added it in as the `CommCareSupportLibrary` dependency is throwing an exception if this dependency is not added to the biometric application. 

Any thoughts on why that is happening? My understanding is that since the dependency is in `CommCareSupportLibrary` we don't need it in the biometric application.

Reply:
Well, there is the fact the we are now adding the `aar` of the support library to the project, which, if I'm not mistaken,  causes all the transitive dependencies to be dropped, so you only get the library. However, this is temporary, once we release a new version of the support library and add it like any other dependency, this should be fine.

Reply:
Ah, I wasn't aware this was happening. That makes sense, I'll remove the unnecessary dependency then.